### PR TITLE
docs: add github action in CI story

### PR DESCRIPTION
**Pull request by shhdharmen**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

Add guideline to configure project for Github Actions

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by gkalpak — COMMENTED** [review]
Drive-by comment:

Please change the commit message to `docs: ...`. (`docs-infra` refers to changes to our documentation tooling.)

**Comment by shhdharmen** [comment]
@googlebot I fixed it.

File: aio/content/guide/testing.md [hunk 1]
```diff
@@ -272,6 +272,37 @@ Step 3: Commit your changes and push them to your repository.
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: npm install and test
```
Comment (josephperrott) [inline aio/content/guide/testing.md:293]: I would recommend separating this into two steps `setup` and `test` to match how we set up things for our other CI provider examples.

**Review by josephperrott — APPROVED** [review]
LGTM, from a content engineering content perspective.

**Comment by shhdharmen** [comment]
FYI, provided workflow for GitHub Actions ran successfully the other day: [link](https://github.com/shhdharmen/test-gh-actions/runs/2332301717?check_suite_focus=true).

But, as of today, it's failing: [link](https://github.com/shhdharmen/test-gh-actions/runs/2341146783?check_suite_focus=true)

For a successful run, it used chromedriver version **89.0.4389.23** (with chrome version **89.0.4389.114**) and when it failed, it used version **90.0.4430.24** (with chrome version **90**)

So, as of now, chromedriver version **90.0.4430.24** only supports chrome version **90**. And it's not available as [stable version](https://www.ubuntuupdates.org/package/google_chrome/stable/main/base/google-chrome-stable).

**Comment by shhdharmen** [comment]
So today workflow ran successfully, [link](https://github.com/shhdharmen/test-gh-actions/runs/2351203268). Seems like there was delay on updating stable version for ubuntu.

**Review by AndrewKushnir — APPROVED** [review]

**Pull request closed without merging**